### ISSUE TITLE
Problems with the scannetv2 dataset

### ISSUE THREAD

**Issue by XIAOGUOY** [dvlab-research/Stratified-Transformer#60]
The following issues occur while the scannetv2 dataset is running and do not know how to solve them.
![7bf6bfbe77e57cdb74f3272b7eb88d2](https://user-images.githubusercontent.com/90907941/192729380-7dcc4e36-87b0-46e7-bdb2-405c60861a8e.jpg)

**Comment by X-Lai** [comment]
Please check your data_path.

**Comment by XIAOGUOY** [comment]
> 请检查您的data_path。

Is there anything else I need to change the path to in addition to the .yaml file?

**Comment by X-Lai** [comment]
No, you only need to modify the .yaml file. But it seems that the data_path you entered is not eligible.
